src/sql/azdata.proposed.d.ts

```diff
@@ -1802,7 +1802,7 @@ declare module 'azdata' {
 
 	export interface TextComponentProperties {
 		/**
-		 * Corresponds to the aria-live accessibility attribute for this component
+		 * Corresponds to the aria-live accessibility attribute for this component. The accepted values are polite, assertive, and off
```

Comment:
could you please add a new union type?

Reply:
And you can remove the added text when you do that since it'll be clear enough with the specific type